docs/containers/bridge-to-kubernetes.md

```diff
@@ -83,47 +81,47 @@ az aks get-credentials --resource-group MyResourceGroup --name MyAKS
 
 > **Important**: You can only redirect services that have a single pod.
 
 After you select your service, you are prompted to enter the TCP port for your local application. For this example, enter "3000".
 
-![Connect choose port](images/local-process-kubernetes-vs-code/choose-port.png)
+![Connect choose port](images/bridge-to-kubernetes-vs-code/choose-port.png)
 
 Choose **Launch via NPM** as the launch task.
 
-![Connect choose launch task](images/local-process-kubernetes-vs-code/choose-launch.png)
+![Connect choose launch task](images/bridge-to-kubernetes-vs-code/choose-launch.png)
 
 > **Note**: You will be prompted to allow the **EndpointManager** to run elevated and modify your hosts file.
 
-You have the option of running isolated or not isolated. If you run isolated, only your requests are routed to your local process; other developers can use the cluster without being affected. If you don't run isolated, all traffic is redirected to your local process. For more information on this option, see [Using routing capabilities for developing in isolation](https://docs.microsoft.com/visualstudio/containers/overview-local-process-kubernetes#using-routing-capabilities-for-developing-in-isolation).
+You have the option of running isolated or not isolated. If you run isolated, only your requests are routed to your local process; other developers can use the cluster without being affected. If you don't run isolated, all traffic is redirected to your local process. For more information on this option, see [Using routing capabilities for developing in isolation][btk-overview-routing].
 
-![Isolation prompt](images/local-process-kubernetes-vs-code/lpk-isolation-prompt.png)
+![Isolation prompt](images/bridge-to-kubernetes-vs-code/btk-isolation-prompt.png)
 
 Your development computer is connected when the VS Code status bar turns orange and the Kubernetes extension shows you are connected.
 
-![Development computer connected](images/local-process-kubernetes-vs-code/development-computer-connected.png)
+![Development computer connected](images/bridge-to-kubernetes-vs-code/development-computer-connected.png)
 
-> **Note**: On subsequent launches, you will not be prompted for the service name, port, launch task, or whether to run isolated. These values are saved in `.vscode/tasks.json`. To change these settings later, open the Command Palette, and run the command **Local Process with Kubernetes: Configure Local Process with Kubernetes**.
+> **Note**: On subsequent launches, you will not be prompted for the service name, port, launch task, or whether to run isolated. These values are saved in `.vscode/tasks.json`. To change these settings later, open the Command Palette, and run the command **Bridge to Kubernetes: Configure Bridge to Kubernetes**.
```

Comment:
Minor comment: the command has been renamed recently, and is now called `Bridge to Kubernetes: Configure` rather than `Bridge to Kubernetes: Configure Bridge to Kubernetes`.